flights/berks_ceh_hh_epi_services_flight.yaml

```diff
@@ -0,0 +1,380 @@
+  #     ol.type: 
+  #       type: "ol.type"
+  #       column: "ProjectType"
+  #       transforms:
+  #       - !<transforms.ParseIntTransform> {}
+  #       - !<transforms.ReplaceTransform>
+  #         target: ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14"]
+  #         goal: ["Emergency Shelter", "Transitional Housing", "PH - Permanent Supportive Housing", "Street Outreach", "Services Only", "Other", "Safe Haven", "PH – Housing Only", "PH – Housing with Services (no disability required for entry)", "Day Shelter", "Homelessness Prevention", "PH - Rapid Re-Housing", "Coordinated Assessment"]
+  #     ol.description: 
+  #       type: "ol.description"
+  #       column: "HousingType"
+  #       transforms:
+  #       - !<transforms.ParseIntTransform> {}
+  #       - !<transforms.ReplaceTransform>
+  #         target: ["1", "2", "3"]
+  #         goal: ["site-based - single site", "site-based - clustered/multiple sites", "tenant-based - scattered site"]
+  #     ol.datetimestart: 
+  #       type: "ol.datetimestart"
+  #       column: "OperatingStartDate"
+  #       transforms:
+  #       - !<transforms.DateTimeTransform>
+  #         pattern: ["yyyy-MM-dd HH:mm:ss"]
+  #     ol.datetimeend: 
+  #       type: "ol.datetimeend"
+  #       column: "OperatingEndDate"
+  #       transforms:
+  #       - !<transforms.DateTimeTransform>
+  #         pattern: ["yyyy-MM-dd HH:mm:ss"]
+  #   name: "program"
+
+
+  service:
+    fqn: "ol.encounter" 
+    entitySetName: "BerksCEHServices"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id" 
+        column: "ServicesID"
+      ol.datetimestart:
+        type: "ol.datetimestart" 
+        column: "DateProvided"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+      ol.servicetype: 
+        type: "ol.servicetype"
+        column: "RecordType"
+        transforms:
+        - !<transforms.ReplaceTransform>
+          target: ["12", "13", "141", "142", "143", "144", "151", "152", "161", "162", 200", "210"]
+          goal: ["Contact", "Contact", "PATH Service", "RHY Service", "HOPWA Service", "SSVF Service", "HOPWA financial assistance", "SSVF Financial Assistance", "PATH referral", "RHY referral", "Bed Night", "HUD-VASH OTH voucher tracking"]
+      ol.type3:
+        type: "ol.type"
+        transforms:
+        - !<transforms.BooleanContainsTransform> #not sure if i can do this. 
+          column: "RecordType"
+          string: "144"
+          transformsIfTrue:
+            column: "TypeProvided"
+          - !<transforms.ReplaceTransform>
+            target: ["1", "2", "3", "4", "5", "6"] 
+            goal: ["Outreach services", "Case management services", "Assistance obtaining VA benefits", "Assistance obtaining/coordinating other public benefits", "Direct provision of other public benefits", "Other (non-TFA) supportive service approved by VA"]
+      ol.reason:
```

Comment:
Okay, so there was a lot being written into "type" which seems like a catchall property - would end up having tons of multiplicity. I broke one of these out into "reason" to get some separation, and it seemed like it fit the closest.